GOVERNANCE.md

```diff
@@ -0,0 +1,225 @@
+
+### Maintainers
+
+Maintainers are elected Contributors who serve on one or more Teams.
+For the election process, see Membership Changes below.
+
+Maintainers are expected to:
+
+- enable and promote Flux community Values
+- engage with end Users through appropriate communication channels appropriate to their teams
+- serve as a point of conflict resolution between Contributors to the Flux area appropriate to their teams
+- maintain open collaboration within and across teams
+- additional responsibilites are defined per team below
+
+## Teams
+
+The Flux project is governed by Maintainer teams, each with defined scope and resonsibilities.
+Teams are responsible for the overall organization, code development, community management, and security.
+
+The goal of governing by teams is to allow parts of the project to act independently as appropriate, enabling the project to grow horizontally while remaining well-maintained.
+
+Teams below are also mapped to GitHub teams within the `fluxcd` GitHub org.
+Their members are mapped to GitHub user accounts.
+
+### Org team
+
+@fluxcd/org <https://github.com/orgs/fluxcd/teams/org>
+
+Scope and responsibilities:
+
+- shared Maintainers responsibilies, as well as
+- overseeing the project health and growth
+- maintaining the brand, mission, vision, values, and scope of the overall project
+- changes to licencing and intellectual property
+- administering access to all project assets
+- handling code of conduct violations
+- managing financial decisions
+- defining the scope and responsibilities of all other teams
+- making descisions that affect multiple teams
+- resolving escalated team decisions when the teams responsible are blocked
+- changes to governance (this document)
+
+### Dev team
+
+@fluxcd/dev <https://github.com/orgs/fluxcd/teams/dev>
+
+Scope and responsibilities:
+
+- shared Maintainers responsibilies, as well as
+- all technical decisions related to project software
+- administering, creating, archiving, and deleting source code repositories
+- release management
+
+In future, the scope and responsibilities of this team may be split into separate dev-related teams.
+But even if so, due to Flux componentized architecture, there should be no assumption of a 1:1 relationship between a source repository and a team.
+
+### Security team
+
+@fluxcd/security <https://github.com/orgs/fluxcd/teams/security>
+
+Scope and responsibilities:
+
+- shared Maintainers responsibilies, as well as
+- point of contact for reporting security vulnerabilities
+- responding to and investigating security reports
+- patching and releasing security fixes
+- publishing the vulnerability disclosure once a mitigation strategy is available
+- documenting overall project security best practices
+
+Anyone may email this team privately to safely raise possible security vulnerabilities <https://lists.cncf.io/g/cncf-flux-community>.
```

Comment:
Yes thanks nice catch. Security team was intended to have its own email. This is critical. It should be set up not as a discussion list, because it must necessarily be a private channel for vulnerability reports.